x-pack/metricbeat/module/aws/cloudwatch/cloudwatch.go

```diff
@@ -25,11 +25,13 @@ import (
 
 var (
 	metricsetName          = "cloudwatch"
-	metricNameIdx          = 0
-	namespaceIdx           = 1
-	statisticIdx           = 2
-	identifierNameIdx      = 3
-	identifierValueIdx     = 4
+	accountIdIdx           = 0
+	accountLabelIdx        = 1
+	metricNameIdx          = 2
+	namespaceIdx           = 3
+	statisticIdx           = 4
+	identifierNameIdx      = 5
+	identifierValueIdx     = 6
```

Comment:
Related to Tom's comment above, I'd move this stuff out to a separate struct/type with all the information needed to compose labels. We can address it in another PR.